Source/Charts/Charts/PieRadarChartViewBase.swift

```diff
@@ -555,11 +555,10 @@ open class PieRadarChartViewBase: ChartViewBase
             
             if !rotationWithTwoFingers
             {
-                let touch = touches.first as NSUITouch!
-                
-                let touchLocation = touch?.location(in: self)
-                
-                processRotationGestureBegan(location: touchLocation!)
+                if let touch = touches.first {
```

Comment:
I'm not sure why use if here. Any ideas?

Reply:
updated